lms/djangoapps/courseware/features/problems.feature

```diff
@@ -108,14 +108,10 @@ Feature: LMS.Answer problems
         When I answer a "multiple choice" problem "correctly"
         Then The "Reset" button does not appear
 
-    Scenario: I can view and hide the answer if the problem has it:
+    Scenario: I can view the answer if the problem has it:
         Given I am viewing a "numerical" that shows the answer "always"
         When I press the button with the label "Show Answer"
-        Then the Show/Hide button label is "Hide Answer"
```

Comment:
Are we replacing this removed testing with something else? Or is it no longer valid?